Makefile: Fix 'make update bin/eclair.jar'

It is currently not working.

To reproduce: 

	make docker-build ; docker run lnintegration bash -c "make update-eclair bin/eclair.jar"

Error:

```
Downloading from central: https://repo.maven.apache.org/maven2/org/scala-lang/scala-library/2.11.6/scala-library-2.11.6.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/scala-lang/scala-library/2.11.6/scala-library-2.11.6.jar (5.6 MB at 16 MB/s)
[ERROR] /root/lightning-integration/src/eclair/eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala:19: error: object CRC32C is not a member of package java.util.zip
[ERROR] import java.util.zip.CRC32C
[ERROR]        ^
[ERROR] /root/lightning-integration/src/eclair/eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala:1059: error: not found: type CRC32C
[ERROR]     val checksum = new CRC32C()
[ERROR]                        ^
[ERROR] two errors found
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for eclair_2.11 0.3.2-SNAPSHOT:
[INFO] 
[INFO] eclair_2.11 ........................................ SUCCESS [ 12.144 s]
[INFO] eclair-core_2.11 ................................... FAILURE [ 58.491 s]
[INFO] eclair-node ........................................ SKIPPED
[INFO] eclair-node-gui .................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:10 min
[INFO] Finished at: 2019-08-29T15:23:10Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal net.alchim31.maven:scala-maven-plugin:3.4.2:compile (scalac) on project eclair-core_2.11: wrap: org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :eclair-core_2.11
cp src/eclair/eclair-node/target/eclair-node-*-d67ba48.jar bin/eclair.jar
cp: cannot stat 'src/eclair/eclair-node/target/eclair-node-*-d67ba48.jar': No such file or directory
make: *** [bin/eclair.jar] Error 1
Makefile:40: recipe for target 'bin/eclair.jar' failed
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -32,7 +32,7 @@ RUN apt-get update \
     libzmq3-dev \
     miniupnpc \
     net-tools \
-    openjdk-8-jdk \
+    openjdk-11-jdk \
     pkg-config \
     python \
     python3 \
@@ -59,6 +59,7 @@ RUN cd /tmp \
     && cp bitcoin-$BITCOIN_VERSION/bin/bitcoin* /usr/bin/ \
     && rm -rf $BITCOIN_TARBALL bitcoin-$BITCOIN_VERSION
 
+# maven for java builds (eclair)
 RUN cd /tmp \
     && wget -qO mvn.tar.gz https://www-us.apache.org/dist/maven/maven-3/3.6.0/binaries/apache-maven-3.6.0-bin.tar.gz \
     && tar -xzf mvn.tar.gz \
@@ -74,9 +75,6 @@ RUN cd /tmp \
     && ln -s /usr/local/go/bin/go /usr/bin/
 
 ENV GOROOT=/usr/local/go
-
-# eclair
-RUN update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
 
 VOLUME /root/lightning-integration/reports
 VOLUME /root/lightning-integration/output

